src/pybind/mgr/dashboard_v2/frontend/src/defaults.scss

```diff
@@ -1 +1,10 @@
 $oa-color-blue: #288cea;
+$oa-color-light-blue: #afd9ee;
+$bg-light-blue: #d9edf7;
```

Comment:
I suggest  to use ``$bg-color-light-blue`` or ``$bg-light-blue-color`` to make it clear that this is a color.